endToEnd/endToEndTest.sh

```diff
@@ -53,11 +53,12 @@ dcm2pnm ./endToEnd/level-5-frames-0-1.dcm ./endToEnd/test4GeneratedImage.ppm
 diff ./endToEnd/test4GeneratedImage.ppm ./endToEnd/test4ExpectedImage.ppm
 
 #test - generate bilinear Downsamped DICOM and compare with expected image
-./build/wsi2dcm $fileName ./endToEnd/ --seriesDescription test5  --levels 6 --startOn 5 --tileHeight 100  --tileWidth 100 --compression raw --BilinearDownsampeling
+./build/wsi2dcm $fileName ./endToEnd/ --seriesDescription test5  --levels 6 --startOn 5 --tileHeight 100  --tileWidth 100 --compression raw --bilinearDownsampeling
```

Comment:
spelling mistake, should be "Downsampling"

Reply:
fixed

Reply:
Ah thanks for the fix, but my comment was actually referring to the BilinearDownsampeling flag itself. =P